doc/whats_new.rst

```diff
@@ -6,8 +6,11 @@
  What's New
 ============
 
-TBD (TBD)
-=========
+v2.20.0.0 (15. Aug 2019)
+========================
+- Update Stan source to v2.20.0 (`release notes <https://github.com/stan-dev/stan/releases/tag/v2.20.0>`_)
+- Fix: Use prefix 'pystan_' with tempfolder (@ahartikainen)
+- Reminder: Python 2 is not supported after 2019-12-31. PyStan will phase out testing of Python 2 after this point.
```

Comment:
- Convenience functions for external C++ (advanced users)
- possibility to set inv_metric manually (+ convenience functions to get stepsize, inv_metric and last_location/last_warmup)

Reply:
- Experimental function to load old pickled fits, without the proper model.

Reply:
You're right. I missed those. I'll add them.